src/views/index.ts

```diff
@@ -4,7 +4,7 @@ export { default as About } from "./About";
 export { default as Confirmation } from "./Confirmation";
 export { default as AllTransactions } from "./Transactions/allTransactions";
 export { default as MyTransactions } from "./Transactions/myTransactions/";
-export { default as Rewards } from "./Rewards";
+export { default as Rewards } from "./Referrals";
```

Comment:
nit: rename to fit the file name? and also `Rewards` also should be exported I guess
```suggestion
export { default as Referrals } from "./Referrals";
export { default as Rewards } from "./Rewards";
```